src/imgui/ImGuiReverseBar.cc

```diff
@@ -206,12 +205,11 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 						});
 					}
 					if (ImGui::TableNextColumn()) {
-						ImGui::TextUnformatted("Preview"sv);
 						ImVec2 size(320, 240);
 						if (previewImage.texture.get()) {
 							ImGui::Image(previewImage.texture.getImGui(), size);
 						} else {
-							ImGui::Dummy(size);
+							ImGui::TextUnformatted("No preview available..."sv);
```

Comment:
The point of this `ImGui::Dummy()` call is to reserve space (as big as the image would be). The effect is that when you move the mouse up/down, so hover over different rows, some with and some without preview, the size of the submenu remains the same. And that might be important when it's close to the window or even screen border, to avoid that the submenu changes position.

So maybe we should keep this? One way to do that is with something like this:
```
auto pos = ImGui::GetCursorPos();
ImGui::Dummy(size);
ImGui::SetCursorPos(pos);
ImGui::TextUnformatted("...");
```
In other words: get and restore the imgui cursor position to draw "overlapping" elements (but one of the elements only takes space but is itself invisible).

Optionally, in the call to `SetCursorPos()` you could also add something to the y-coordinate to make the text appear vertically centered.

Reply:
Totally agreed.... See update where I address it and center it nicely in the box. Perhaps the centering could be put in ImGuiUtils or something. Not sure how to do that though, as you need to capture the cursor pos before drawing the thing you want to center in.